### PR TITLE
[badgeth] Badgeth strategy

### PR DESCRIPTION
**Pull request by HardForksOverKnives**
Adds Badgeth strategy

### CODE REVIEW

File: src/strategies/index.ts [hunk 1]
```diff
@@ -219,5 +220,6 @@ export default {
   'aave-governance-power': aaveGovernancePower,
   cake,
   'planet-finance': planetFinance,
+  'badgeth': badgeth
```
Comment (ChaituVR) [inline src/strategies/index.ts:223]:
```suggestion
  'badgeth': badgeth,
```

**Review by ChaituVR — COMMENTED** [review]
LGTM

**Review by bonustrack — CHANGES_REQUESTED** [review]
The strategy should return only checksumed addresses (not lowercased addrrsses), you can checksum address with This function `import { getAddress } from '@ethersproject/address';`.

**Comment by HardForksOverKnives** [comment]
Checksummed addresses

File: src/strategies/badgeth/index.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  };
+  if (snapshot !== 'latest') {
+    // @ts-ignore
+    params.voters.__args.block = { number: snapshot };
+  }
+
+  const score = {};
+  console.log("sending subgraph request");
+  const result = await subgraphRequest(BADGETH_SUBGRAPH_URL, params);
+  console.log("result: ", result);
```
Comment (bonustrack) [inline src/strategies/badgeth/index.ts:40]: You shouldn't add console.log in the strategy
```suggestion
```

File: src/strategies/badgeth/index.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      votingPower: true
+    }
+  };
+  if (snapshot !== 'latest') {
+    // @ts-ignore
+    params.voters.__args.block = { number: snapshot };
+  }
+
+  const score = {};
+  console.log("sending subgraph request");
```
Comment (bonustrack) [inline src/strategies/badgeth/index.ts:38]:
```suggestion
```

**Review by bonustrack — APPROVED** [review]

**Pull request merged**